app/assets/javascripts/application.js

```diff
@@ -66,10 +66,30 @@ var checkboxSelector = (function() {
   return {
     init: cs.init
   }
-})();
+})()
+
+var yesNoButtons = (function () {
+  var yn = {
+    init: function () {
+      $('[data-no-path]').on('click', function () {
+        $('input.boolean-answer').val('0')
+        $('#step_redirect_path').val($(self).data('no-path'))
+      })
+
+      $('[data-yes-path]').on('click', function () {
+        $('input.boolean-answer').val('1')
+        $('#step_redirect_path').val($(self).data('yes-path'))
+      })
+    }
+  }
+  return {
+    init: yn.init
+  }
+})()
 
-$(document).ready(function() {
-  radioSelector.init();
-  checkboxSelector.init();
-  textWell.init();
-});
+$(document).ready(function () {
+  radioSelector.init()
+  checkboxSelector.init()
+  textWell.init()
+  yesNoButtons.init()
+})
```

Comment:
for what it's worth, my vim linters did a number on this file, so - the important parts are everything related to `yesNoButtons`